docs/edge.md

```diff
@@ -1,92 +1,141 @@
+We raise the following question[^1]:
 
-The question is: How do you calculate that? How do you know if you wanna play?
+!!! Question "Which trade is a better option?"
+    a) A trade with 80% of chance of losing $100 and 20% chance of winning $200<br/>
+    b) A trade with 100% of chance of losing $30
 
-The answer comes to two factors:
+??? Info "Answer"
+    The expected value of *a)* is smaller than the expected value of *b)*.<br/>
+    Hence, *b*) represents a smaller loss in the long run.<br/>
+    However, the answer is: *it depends*
 
-- Win Rate
-- Risk Reward Ratio
+Another way to look at it is to ask a similar question:
 
-### Win Rate
+!!! Question "Which trade is a better option?"
+    a) A trade with 80% of chance of winning 100 and 20% chance of losing $200<br/>
+    b) A trade with 100% of chance of winning $30
 
-Win Rate (*W*) is is the mean over some amount of trades (*N*) what is the percentage of winning trades to total number of trades (note that we don't consider how much you gained but only if you won or not).
+Edge positioning tries to answer the hard questions about risk/reward and position size automatically, seeking to minimizes the chances of losing of a given strategy.
 
-```
-W = (Number of winning trades) / (Total number of trades) = (Number of winning trades) / N
-```
+### Trading, winning and losing
 
-Complementary Loss Rate (*L*) is defined as
+Let's call $o$ the return of a single transaction $o$ where $o \in \mathbb{R}$. The collection $O = \{o_1, o_2, ..., o_N\}$ is the set of all returns of transactions made during a trading session. We say that $N$ is the cardinality of $O$, or, in lay terms, it is the number of transactions made in a trading session.
 
-```
-L = (Number of losing trades) / (Total number of trades) = (Number of losing trades) / N
-```
+!!! Example
+    In a session where a strategy made three transactions we can say that $O = \{3.5, -1, 15\}$. That means that $N = 3$ and $o_1 = 3.5$, $o_2 = -1$, $o = 15$.
 
-or, which is the same, as
+A winning trade is a trade where a strategy *made* money. Making money means that the strategy closed the position in a value that returned a profit, after all deducted fees. Formally, a winning trade will have a return $o_i > 0$. Similarly, a losing trade will have a return $o_j \leq 0$. With that, we can discover the set of all winning trades, $T_{win}$, as follows:
 
-```
-L = 1 – W
-```
+$$ T_{win} = \{ o \in O | o > 0 \} $$
+
+Similarly, we can discover the set of losing trades $T_{lose}$ as follows:
+
+$$ T_{lose} = \{o \in O | o \leq 0\} $$
+
+!!! Example
+    In a section where a strategy made three transactions $O = \{3.5, -1, 15, 0\}$:<br>
+    $T_{win} = \{3.5, 15\}$<br>
+    $T_{lose} = \{-1, 0\}$<br>
+
+### Win Rate and Lose Rate
+
+The win rate $W$ is the proportion of winning trades with respect to all the trades made by a strategy. We use the following function to compute the win rate:
+
+$$W = \frac{\sum^{o \in T_{win}} o}{N}$$
```

Comment:
Are you sure this (and the Corresponding L formulae) are correct?
the sum symbol seems odd to me - it's not the sum of trades (`sum(winning_trades)`) - but the count (`len(winning_trades)`) that's calculated here, especially considering that the same "upper part" is used for average_profit - but there it's really the sum of profits - not the count of trades.

But i'm no mathematician, so this syntax is ... odd for me.

Reply:
You are absolutely correct. I had made `o` as a logical notation before, but noticed that summations were used further in the document. So with a logical notation it would be `o*v` where `o \in {0,1}`, but I figured it would be confusing to the user. I have changed the formulas to reflect the size of the sets.